src/txn/txn_recover.c

```diff
@@ -741,13 +742,15 @@ __wt_txn_recover(WT_SESSION_IMPL *session, const char *cfg[])
     WT_RECOVERY_FILE *metafile;
     char *config;
     char ts_string[2][WT_TS_INT_STRING_SIZE];
-    bool do_checkpoint, eviction_started, hs_exists, needs_rec, was_backup;
+    bool do_checkpoint, eviction_started, hs_exists, needs_rec, was_backup, rts_executed,
```

Comment:
Nit: We usually don't let declarations wrap like that.
Start another line like:
```
bool a, b, c, ...;
bool x, y, z;
```

Reply:
Thanks for pointing this out. Fixed.